test/02_USM_internal.test.js

```diff
@@ -38,28 +38,28 @@ contract('USM - Internal functions', (accounts) => {
 
   describe('functionality', async () => {
     it('returns the oracle price in WAD', async () => {
-      let oraclePrice = (await mockToken.oraclePrice())
+      let oraclePrice = (await usm.oraclePrice())
       oraclePrice.toString().should.equal(priceWAD.toString())
     })
 
     it('returns the value of eth in usm', async () => {
       const oneEth = WAD
       const equivalentUSM = oneEth.mul(priceWAD).div(WAD)
-      let usmAmount = (await mockToken.ethToUsm(oneEth))
+      let usmAmount = (await usm.ethToUsm(oneEth))
       usmAmount.toString().should.equal(equivalentUSM.toString())
     })
 
     it('returns the value of usm in eth', async () => {
       const oneUSM = WAD
       const equivalentEth = oneUSM.mul(WAD).div(priceWAD)
-      let ethAmount = (await mockToken.usmToEth(oneUSM.toString()))
+      let ethAmount = (await usm.usmToEth(oneUSM.toString()))
       ethAmount.toString().should.equal(equivalentEth.toString())
     })
 
-    it('returns the debt ratio as zero', async () => {
-      const ZERO = new BN('0')
-      let debtRatio = (await mockToken.debtRatio())
-      debtRatio.toString().should.equal(ZERO.toString())
+    it('returns the debt ratio as MAX_DEBT_RATIO', async () => {
+      const MAX_DEBT_RATIO = await usm.MAX_DEBT_RATIO()
```

Comment:
I guess maybe put ()s around the `await` like everywhere else I dunno

Reply:
Better to remove them everywhere else!